document/core/appendix/algorithm.rst

```diff
@@ -25,7 +25,7 @@ Types are representable as an enumeration.
 
 .. code-block:: pseudo
 
-   type val_type = I32 | I64 | F32 | F64 | V128 | Funcref | Externref
+   type val_type = I32 | I64 | F32 | F64 | V128 | Funcref | Exnref | Externref
```

Comment:
how about `is_ref` below? it seems unused though.

Reply:
Indeed, case added.